lib/logasm/preprocessors/whitelist.coffee

```diff
@@ -1,15 +1,26 @@
 {isObject} = require('../object_helpers')
 
+isWildcardPointer = (pointer) ->
```

Comment:
this flow is later replaced by a hashmap lookup https://github.com/salemove/node-logasm/pull/7/commits/6ea8b81044d6c83ce534a73c4381d3854ce71431 . probably no need to dive deep into this commit.